packages/core/src/components/forms/_form-group.scss

```diff
@@ -0,0 +1,118 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+@import "../../common/mixins";
+
+/*
+Form group
+
+Add additional helper text to help users navigate forms and complex fields.
+
+<div class="pt-callout pt-intent-primary pt-icon-info-sign">
+  Note that this feature necessitates additional DOM elements,
+  thus requiring a more complex form structure achieved with `.pt-form-group`.
+</div>
+
+- Link each label to its respective control element with a `for={#id}` attribute on the `<label>` and
+`id={#id}` on the control.
+
+- Add extra information to the label with `span.pt-text-muted`.
+
+- Add an element with `.pt-form-helper-text` to reflect user input (display intent with `.pt-intent-*`).
+
+- Add `.pt-inline` to `.pt-form-group` to place the label to the left of the control.
+
+- Add `.pt-large` to `.pt-form-group` to align the label when used with large Blueprint controls.
+
+Markup:
+<div class="pt-form-group">
+  <label for="example-form-group-input-a">
```

Comment:
API question: why do these labels not use `.pt-label`?

this section in the docs immediately follows "Labels" where we seem to instruct users to write `label.pt-label`, but suddenly we're not following our own rules.